server/portal/apps/projects/views.py

```diff
@@ -80,60 +85,26 @@ def get(self, request):
         }
         ```
         """
-        query_string = request.GET.get('query_string')
-        offset = int(request.GET.get('offset', 0))
-        limit = int(request.GET.get('limit', 100))
-
-        mgr = ProjectsManager(request.user)
-
-        if query_string is not None:
-            res = mgr.search(query_string=query_string,
-                             offset=offset,
-                             limit=limit)
-        else:
-            res = mgr.list(
-                offset=offset,
-                limit=limit
-            )
-        return JsonResponse(
-            {
-                'status': 200,
-                'response': res
-            },
-            encoder=ProjectsManager.systems_serializer_cls
-        )
+        # query_string = request.GET.get('query_string')
+        # offset = int(request.GET.get('offset', 0))
+        # limit = int(request.GET.get('limit', 100))
```

Comment:
Can we drop these lines?

Reply:
I actually do want to keep these since we'll need them for search in the future. I'll make a TODOV3 note to explain.

Reply:
@jarosenb Can you add a TODOv3 note here?